tests/ea/mutation/mutator/test_mutate.py

```diff
@@ -49,7 +49,9 @@ def _test_mutate(mutator, record, mutation_record):
     assert (
         result.get_mutator_name() == mutation_record.get_mutator_name()
     )
+    original_mol = result.get_molecule_record().get_molecule()
```

Comment:
`original_molecule`
`mutated_molecule`
and append the `with_canoncical_atom_ordering` to it too.